src/components/Ingredients.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
+import Ingredient from './Ingredient';
 
-const Ingredients = () => (
-  <div className="ingredients">Here is an ingredients list</div>
-)
+class Ingredients extends React.Component {
+
+  extractIngredient = (ingredientsList) => {
+    console.log(ingredientsList);
+
+    return Object.keys(ingredientsList).map(function(key){
+      const ingObj = ingredientsList[key];
+      return <Ingredient
+        key={ingObj['ingredient']}
+        amount={ingObj['amount']}
+        measure={ingObj['measure']}
+        ingredient={ingObj['ingredient']} />
+      })
+    };
+
+
+  render() {
+
+    // console.log(this.props);
```

Comment:
There's some console.logs that are commented out and others that aren't. You should remove those before merging.